thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -229,14 +229,13 @@ export default Component.extend({
   showRules: computed(
     'isPreviewMode',
     'granularity',
-    'dimensionExploration',
     function() {
       const {
         isPreviewMode,
         granularity,
         dimensionExploration
-      } = this.getProperties('isPreviewMode', 'granularity', 'dimensionExploration');
-      return (isPreviewMode || (!dimensionExploration && ((granularity || '').includes('DAYS'))));
+      } = this.getProperties('isPreviewMode', 'granularity');
```

Comment:
I am hesitate to remove the restriction on "dimensionExploration" which may cause lots of computations.
How about just opening to minute level metrics w/o dimension exploration?

Reply:
Sure, I will make those changes.

Reply:
Updated